uiowa/uiowa
# Alerts a11y improvements.

Resolves #1898 

Related to #1899

### Testing
- Check out main.
- `blt sync`
- `drush cset uiowa_alerts.settings hawk_alert.source 'https://emergency.stage.drupal.uiowa.edu/api/active.json'`
- Run SiteImprove locally on default site on main branch. 
- See ARIA landmark errors regarding perceivable content in Hawk Alerts.
- Check out this branch.
- `yarn workspace uiowa_alerts build`
- Refresh page and run SiteImprove report again.
- See errors are not present and `hawk-alert-message` divs have ARIA role and label.
